MTB_SCORE/score/get_syndata.py

```diff
@@ -0,0 +1,80 @@
+# MTB Scoring
+
+# import the required packages
+import synapseclient
+from pyarrow import fs
+import pyarrow.parquet as pq
+
+import pandas as pd
+import os
+import json
+import logging
+
+logging.basicConfig(level=logging.INFO)
+logger=logging.getLogger()
+
+def get_synclient(USER_NAME, PASS):
+    """
+    -----------------------------------------------------------------------------------------
+    
+    Get synapse credential
+    
+    Args:
+        USER_NAME: Username
+        PASS: Password
+        
+    Returns:
+        results: synapse object
+        
+    -----------------------------------------------------------------------------------------
+    """
+    syn = synapseclient.login(USER_NAME, PASS)
+    return syn
+
+def get_token(PROJECT_ID, syn):
+    """
```

Comment:
Function does not get token, it gets S3filesystem and path.

name should be get_S3_fs